chaos-tests/src/test/java/ChaosIT.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
This one should be changed

Reply:
oh, while reviewing I thought it's a new license header. I will create a follow up PR to clean this up.